tests/main.cpp

```diff
@@ -0,0 +1,17 @@
+#include <gtest/gtest.h>
+#include <resultset.h>
+#include <string.h>
+#include <string>
+
+using namespace std;
+
+int main(int argc, char **argv)
+{
+    testing::InitGoogleTest(&argc, argv);
+
+    testing::GTEST_FLAG(repeat) = 20;
```

Comment:
You can probably increase this count also if the tests don't take long.

Reply:
It will take longer if count is increased